modules/olm-overview.adoc

```diff
@@ -16,6 +16,20 @@ _Operator Lifecycle Manager_ (OLM) helps users install, update, and manage the l
 .Operator Lifecycle Manager workflow
 image::olm-workflow.png[]
 
-OLM runs by default in {product-title} {product-version}, which aids cluster administrators in installing, upgrading, and granting access to Operators running on their cluster. The {product-title} web console provides management screens for cluster administrators to install Operators, as well as grant specific projects access to use the catalog of Operators available on the cluster.
+OLM runs by default in {product-title} {product-version}, which aids
```

Comment:
> OLM runs by default in {product-title} {product-version}...

Do you need to remove the {product-versions} for ROSA/OSD, which [you did earlier](https://github.com/openshift/openshift-docs/pull/61793/files#diff-29a1ce3da2b0a9fd9c598ead7a70e6bd6afb6a075c822f5d2eeb4dab2f6b953aR15-R20)?